app/js/lib/foxbox/recipes.js

```diff
@@ -62,22 +63,30 @@ export default class Recipe {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       // Get all services but the recipes.
       .then(services => services.filter(service =>
-        service.adapter != 'thinkerbell-adapter' &&
-        service.adapter != 'webpush@link.mozilla.org'
+        service.adapter != 'thinkerbell-adapter'
+        && service.adapter != 'webpush@link.mozilla.org'
+        && service.adapter != 'console@link.mozilla.org'
+
+          // Deactivating IP camera from services with getters
+        && service.adapter != 'ip-camera@link.mozilla.org'
```

Comment:
I always use the style formatting on WebStorms. I fine tuned it to fit my needs, but it's not perfect and in some case it results in weird indentations like here. I'll fix that.